osquery/tables/system/windows/logical_drives.cpp

```diff
@@ -5,8 +5,11 @@
  *  This source code is licensed in accordance with the terms specified in
  *  the LICENSE file found in the root directory of this source tree.
  */
-#include <osquery/tables.h>
 
+#include <Windows.h>
```

Comment:
If you don't need some specific stuff from windows api I'd suggest to include this header via `osquery/utils/system/system.h`. It is going to be much safer because of some predefined macro like `NOMINMAX`.

Reply:
So that's where those JSON warnings were coming from. TIL, thanks!

Reply:
Yes, `windows.h` introduces lots of quite dangerous macro :(